test/unit/process/test_cuda_process_set.cpp

```diff
@@ -147,7 +147,7 @@ void testRandomSystemAddJacobianTerms(std::size_t n_cells, std::size_t n_reactio
     elem = get_double();
 
   auto non_zero_elements = cpu_set.NonZeroJacobianElements();
-  auto builder = SparseMatrixPolicy<double>::create(n_species).number_of_blocks(n_cells).initial_value(100.0);
+  auto builder = SparseMatrixPolicy<double>::create(n_species).number_of_blocks(n_cells).initial_value(0.0);
```

Comment:
Is this change still necessary, now that the CPU and GPU functions do the same thing?

Reply:
I think so. The current implementation of CPU and GPU works only when the initial values of Jacobian matrix are zero. Otherwise, we have to write a separate loop to negate the matrix, like my original GPU implementation.

Reply:
But, isn't this just testing the function that subtracts values from a matrix? I don't think it should care if it starts at 0, or 100, or -100. If the CPU and GPU functions are doing the same thing, they should get the same answer.

Reply:
After a second thought, I agree with you that for this test, it should not matter what the initial value is for the Jacobian matrix. We just need to make sure that when running the whole solver, the initial value of Jacobian matrix must be zero to function properly.

Reply:
Done.